developer-wsl-info.md

```diff
@@ -0,0 +1,68 @@
+# Develop in Millicast SDK with Windows Subsystem for Linux (WSL)
+
+## Table of Contents
+- Millicast SDK for WSL
+  - [Running test](#running-test)
+  
+## Running Test
+After completing the general Millicast SDK setup instructions listed in the [developer-info.md](developer-info.md) file, WSL users may experience multiple issues when trying to run test locally.
+
+### First Issue
+ The first issue you may encounter is a SyntaxError occuring at the Puppeteer/util/disposable.js file which will appear in your terminal as an error due to an unexpected "??=" token.
```

Comment:
Suggestion : You can make a section here called `pre-requisistes` and have node version 16 listed there (or in developer-info.md)

Reply:
I have implemented this suggestion and added links in the table of contents to quickly navigate to the error by name, which I also updated bason on your suggestion below.